<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-99P843S" failures="0" tests="5" name="com.villeos.Villeos_Web_app.NewTest" time="69.260" errors="1" timestamp="2023-08-22T13:54:04 IST" skipped="1">
  <testcase classname="com.villeos.Villeos_Web_app.NewTest" name="url" time="26.670"/>
  <testcase classname="com.villeos.Villeos_Web_app.NewTest" name="url" time="0.000">
    <skipped/>
  </testcase> <!-- url -->
  <testcase classname="com.villeos.Villeos_Web_app.NewTest" name="url" time="4.612"/>
  <testcase classname="com.villeos.Villeos_Web_app.NewTest" name="Setup" time="0.285">
    <error message="&#039;org.openqa.selenium.manager.SeleniumManagerOutput$Result org.openqa.selenium.remote.service.DriverFinder.getPath(org.openqa.selenium.remote.service.DriverService, org.openqa.selenium.Capabilities)&#039;" type="java.lang.NoSuchMethodError">
      <![CDATA[java.lang.NoSuchMethodError: 'org.openqa.selenium.manager.SeleniumManagerOutput$Result org.openqa.selenium.remote.service.DriverFinder.getPath(org.openqa.selenium.remote.service.DriverService, org.openqa.selenium.Capabilities)'
at org.openqa.selenium.safari.SafariDriver.generateExecutor(SafariDriver.java:94)
at org.openqa.selenium.safari.SafariDriver.<init>(SafariDriver.java:83)
at org.openqa.selenium.safari.SafariDriver.<init>(SafariDriver.java:79)
at org.openqa.selenium.safari.SafariDriver.<init>(SafariDriver.java:56)
at com.villeos.Villeos_Web_app.Base_class.Setup(Base_class.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:663)
at org.testng.TestRunner.beforeRun(TestRunner.java:652)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:481)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- Setup -->
  <testcase classname="com.villeos.Villeos_Web_app.NewTest" name="url" time="37.693"/>
</testsuite> <!-- com.villeos.Villeos_Web_app.NewTest -->
